backend/prisma/schema.prisma

```diff
@@ -8,11 +8,40 @@ datasource db {
 }
 
 model User {
-  id       String  @id @default(uuid())
-  email    String  @unique @db.VarChar(80)
-  password String  @db.VarChar(300) // Length of the hashed password
+  id            String         @id @default(uuid())
   /// Full name of a user
-  name     String  @db.VarChar(200)
-  phone    String? @db.VarChar(20)
-  location String?
+  name          String         @db.VarChar(200)
+  sex           Sex
```

Comment:
In the modern world making the `sex` field mandatory can lead to the Twitter scandal :sweat_smile:. So I would like to make it optional too. To make it optional, you can create a new migration. In Prisma philosophy, they don't delete or revert migrations, but just keep adding new ones